test/runtests.jl

```diff
@@ -1,10 +1,12 @@
 using Test
-
-import SimpleBenders
+include("../src/SimpleBenders.jl")
+import .SimpleBenders
 using JuMP
 
-import Clp
-import Cbc
+using Gurobi
```

Comment:
Can we use Highs instead of Gurobi? Overall as a rule of thumb, don't use commercial solvers in tests

Reply:
You are correct! I have checked how to retrieve unbounded extreme rays and successfully replace Gurobi with HiGHS.